packit_service/worker/jobs.py

```diff
@@ -119,24 +119,19 @@ def get_packit_commands_from_comment(comment: str) -> List[str]:
         logger.debug("Empty comment, nothing to do.")
         return []
 
-    cmd_start_index = comment.find(REQUESTED_PULL_REQUEST_COMMENT)
+    REQUESTED_PULL_REQUEST_COMMENT = "/packit"
```

Comment:
This constant is already defined on the top of the file. (That's the usual place where we put constants if not placed in the separate file.)

Reply:
Thanks for catching this!  This is a holdover from the small test bed I created for
testing this in isolation.